indico/modules/events/timetable/templates/display/indico/_contribution.html

```diff
@@ -4,20 +4,26 @@
 {% from 'events/timetable/display/indico/_common.html' import render_speakers, render_references, render_attachments,
                                                               render_notes, render_description, render_time %}
 
-{% macro render_contribution(contrib, event, parent=none, nested=false, hide_end_time=false, timezone=none,
-                             show_notes=false, show_location=false) -%}
+{% macro render_contribution(contrib, event, theme_settings, theme_context, parent=none, nested=false, hide_end_time=false,
```

Comment:
I think the administrative theme calls this macro so it probably needs to be updated.

Reply:
IIRC, the admin theme has its own macros.

Reply:
indeed, nvm